.github/workflows/runtime_plots.py

```diff
@@ -0,0 +1,16 @@
+import matplotlib.pyplot as plt
+import json
+import pandas as pd
```

Comment:
Keep standard library imports and third party imports in separate blocks:

```suggestion
import json

import matplotlib.pyplot as plt
import pandas as pd
```

Reply:
:+1: Good to know!